modules/ROOT/pages/kubernetes/quickstart-analytics-cluster.adoc

```diff
@@ -0,0 +1,168 @@
+The password must be the same as for the primary server.
+If you are using GDS Enterprise Edition, you also need to create a secret with the license file and mount it as the _/licenses_ volume mount.
+For more information on how to create a secret, see xref:/kubernetes/plugins.adoc#install-gds-ee-bloom[Install GDS Enterprise Edition (EE) and Bloom plugins].
+
+[source, yaml]
+----
+neo4j:
+  name: analytics-cluster
+  acceptLicenseAgreement: "yes"
+  edition: enterprise
+  password: my-password
+volumes:
+  data:
+    mode: defaultStorageClass
+  # Define the volume mount for the license file:
+  licenses:
+    disableSubPathExpr: true
+    mode: volume
+    volume:
+      secret:
+        secretName: gds-license
+        items:
+          - key: gds.license
+            path: gds.license
+
+# Set the environment variables to download the plugins:
+env:
+  NEO4J_PLUGINS: '["graph-data-science"]'
+
+# Set the configuration for the plugins directory and the mount for the license file:
+config:
+  gds.enterprise.license_file: "/licenses/gds.license"
+  server.directories.plugins: "plugins"
+
+# Disable the Neo4j load balancer and enable the internal service so that the servers can access each other:
+services:
+  neo4j:
+    enabled: false
+  internals:
+    enabled: true
+
+# Enable the analytics cluster and set the type to secondary:
+analytics:
+  enabled: true
+  type: secondary
+
+----
+======
+=====
+
+For all available options, see xref:kubernetes/configuration.adoc[Customizing a Neo4j Helm chart].
+
+== Install the servers
+
+. Install a single Neo4j server using the _neo4j-primary.yaml_ file, created in the previous section:
++
+[source, bash]
+----
+helm install primary neo4j/neo4j -f /path/to/neo4j-primary.yaml
+----
+. Install the first secondary server using the _secondary-gds.yaml_ file, created in the previous section:
++
+[source, bash]
+----
+helm install gds1 neo4j/neo4j -f /path/to/secondary-gds.yaml
+----
+. Repeat step 2 to deploy a second secondary server.
+Use a different name, for example, _gds2_.
+
+
+== Verify the installation
+
+To verify that the cluster is deployed and running, you can install a load balancer and access Neo4j from the Neo4j Browser.
+
+. Deploy a Neo4j load balancer to the same namespace as the Neo4j cluster:
++
+[source, bash]
+----
+helm install lb neo4j/neo4j-load-balancer --set neo4j.name="analytics-cluster"
+----
+. When deployed, copy the external IP of the LoadBalancer to access Neo4j from an application outside the Kubernetes cluster.
+For more information, see xref:kubernetes/accessing-neo4j.adoc#access-outside-k8s[Applications accessing Neo4j from outside Kubernetes].
+. In a web browser, open the Neo4j Browser at _http://EXTERNAL_IP:7474/browser_ and log in using the password you have configured in your values YAML files.
+. Run the following Cypher query to verify that the cluster is deployed and running:
++
+[source, cypher]
+----
+SHOW SERVERS
+----
```

Comment:
this is missing another CYPHER COMMAND. 'SHOW DATABASES'
it would be good to put a screenshot here and explain what exactly in the output of the above command ensures the setup is complete.

Reply:
It's not a good practice to use screenshots where we can add a code snippet. Could you provide me with the results of this commands?